tests/foreman/cli/test_host.py

```diff
@@ -1105,18 +1105,19 @@ def test_positive_parameter_crud(function_host, target_sat):
         {'host-id': function_host['id'], 'name': name, 'value': value}
     )
     host = target_sat.cli.Host.info({'id': function_host['id']})
-    assert name in host['parameters']
-    assert value == host['parameters'][name]
+    assert name in [param["name"] for param in host['parameters']]
+    assert value in [param["value"] for param in host['parameters']]
```

Comment:
If we are going to continue with these new changes then I would suggest to use single quote for string to maintain consistant pattern
```suggestion
    assert name in [param['name'] for param in host['parameters']]
    assert value in [param['value'] for param in host['parameters']]
```

Reply:
Can we merge these assertions like "(name,value) in [(param['name'], param['value']) for param in host['parameters']]"